### PR TITLE
Rhea: Update Python Framework

### PR DESCRIPTION
**Pull request by ax3l**
Noticed some minor typos and necessary updates after a re-install of the venv.

### CODE REVIEW

**Comment by PrometheusPi** [comment]
@ax3l I will have a look.

File: src/picongpu/submit/titan-ornl/pythonOnRhea.profile.example [hunk 1]
```diff
@@ -36,17 +36,21 @@ source $PROJWORK/$proj/python-venv/rhea/bin/activate
 #
 # now run for setting up the python virtual environment
 #   mkdir -p $PROJWORK/$proj/python-venv
-#   cd $PROJWORK/$proj
+#   cd $PROJWORK/$proj/python-venv
```
Comment (PrometheusPi) [inline src/picongpu/submit/titan-ornl/pythonOnRhea.profile.example]: @ax3l Why did you change the directory?

Comment (ax3l) [inline src/picongpu/submit/titan-ornl/pythonOnRhea.profile.example]: I corrected it, unfortunately the old line was wrong: the command below `virtualenv rhea` creates a new directory `rhea` in the *current* directory which is in all other places assumed to be `$PROJWORK/$proj/python-venv/rhea`.

File: src/picongpu/submit/titan-ornl/pythonOnRhea.profile.example [hunk 2]
```diff
@@ -36,17 +36,21 @@ source $PROJWORK/$proj/python-venv/rhea/bin/activate
...
 #   pip install --upgrade pip
 #   pip install --upgrade wheel
+#   pip install --upgrade numpy
 #
 #   pip install adios
 #   pip install six
 #   pip install matplotlib
+#     # might need an backend change (to `Agg`) in
+#     #   $PROJWORK/$proj/python-venv/rhea/lib/python2.7/site-packages/matplotlib/mpl-data/matplotlibrc
+#   pip install mpld3
```
Comment (ax3l) [inline src/picongpu/submit/titan-ornl/pythonOnRhea.profile.example]: that is mpl for d3.js which is awesome when exporting the data behind an image as a serialization, e.g., to change the zoom later on (in the browser or a new python matplotlib figure object reading it)

**Comment by PrometheusPi** [comment]
@ax3l Thanks for the documentation in the comments.
